File: csharp/isogram/Isogram.cs
using System;

public static class Isogram
{
    public static bool IsIsogram(string word)
    {
        string loweredWord = word.ToLower();
        int[] count = new int[256];
        foreach(char c in word)
        {
            if (++count[c] > 1)
                return false;
        }
        return true;
    }
}

File: haskell/isogram/src/Isogram.hs
module Isogram (isIsogram) where

import Data.Char (isLetter, toLower)
import Data.List (nub)

isIsogram :: String -> Bool
isIsogram str = (length $ nub clean_str) == (length clean_str)
            where clean_str = map toLower $ filter isLetter str
